SPECS/ruby/ruby.spec

```diff
@@ -179,6 +188,11 @@ sudo -u test make test TESTS="-v"
 %{rubygems_dir}/rubygems
 
 %changelog
+* Mon Jul 12 2021 Thomas Crain <thcrain@microsoft.com> - 2.7.2-4
+- Add attribution for parts of the install script taken from Fedora (license: MIT)
```

Comment:
Nit: could you mention the version of Fedora as well?

Reply:
Added a mention of the version here and in the attribution earlier in the spec, thanks!